src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TalismanListener.java

```diff
@@ -233,6 +233,11 @@ public void onItemBreak(PlayerItemBreakEvent e) {
                         break;
                     }
                 }
+
+                // Validate broken item is on offhand.
+                if (e.getBrokenItem().equals(inv.getItemInOffHand())) {
+                    slot = 40;
+                }
```

Comment:
I think this should go *before* the for loop, no point in looping through the armor first.
Pseudo code:

if (offHand) {

}
else {
check armor
}

Reply:
it might be more efficient to also check the main hand, I know the armor slots wouldn't match and then the slot would stay as the main hand value, but it is unnecessary to loop through the armor (4 vs 1 check)
```
if (mainHand) {}
else if (offHand) {}
else {
    loopArmor
}
```